game.libretro/addon.xml.in

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="game.libretro"
        name="Libretro Compatibility"
-       version="2.0.0"
+       version="2.0.1"
        provider-name="Team Kodi">
   <backwards-compatibility abi="1.0.0"/>
   <requires>@ADDON_DEPENDS@</requires>
   <extension point="kodi.gameclient"
        library_@PLATFORM@="@LIBRARY_FILENAME@">
   </extension>
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Libretro compatibility layer for Kodi's Game API</summary>
-    <description>This add-on provides a wrapper for libretro cores, allowing them to be loaded as game add-ons.</description>
+    <summary lang="en_GB">Libretro compatibility layer for Kodi's Game API</summary>
+    <description lang="en_GB">This add-on provides a wrapper for libretro cores, allowing them to be loaded as game add-ons.</description>
     <platform>@PLATFORM@</platform>
-    <nofanart>true</nofanart>
+    <license>GPL-2.0+</license>
```

Comment:
If we're using SPDX like in cpp sources this would be `GPL-2.0-or-later`

Reply:
Ah thanks and sorry, was coming by them https://github.com/kodi-pvr/pvr.hts/pull/447#discussion_r398092817, I change soon.